pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -256,7 +256,7 @@ fn decrease_allowance_works() {
 		// Token does not exist.
 		assert_eq!(
 			decrease_allowance(&addr, 0, &BOB, amount),
-			Err(Module { index: 52, error: [3, 0] }),
+			Err(Module { index: 52, error: [10, 0] }),
```

Comment:
What is changing here?

Reply:
Because we check the `allowance` first before `cancel_approval` called, this returns `Unapproved` instead of `Unknown`